### PR TITLE
:art: Remove Superfluous Comment

### LINKED ISSUE #34: 🎨:art: Remove Superfluous Comment
**Problem**

We have superfluous stuff in our Jenkinsfile; this comes through a copy-paste issue.

**Location**

Here: 

https://github.com/process-engine/process_engine_meta/blob/49b502dffce9493b20b3cfb45ad4bcd8940ac2d7/_templates/Jenkinsfile#L3-L19

and here:

https://github.com/process-engine/process_engine_meta/blob/49b502dffce9493b20b3cfb45ad4bcd8940ac2d7/_templates/Jenkinsfile-node-v7#L3-L19

Those need to be removed.

**Reason:**

This is not very useful.

### PR DESCRIPTION
**Pull request by Paulomart**
## What did you change?

Fixes #34 

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

File: _templates/Jenkinsfile [hunk 1]
```diff
@@ -1,22 +1,5 @@
...
- *  "scripts": {
- *    "lint": "gulp lint",
- *    "build": "gulp build",
- *    "build-schemas": "gulp typescript-schema"
- *    "build-doc": "gulp doc",
- *    "test": "gulp test",
- *  },
- *  (...)
- *
- */
```
Comment (cmg-dev) [inline _templates/Jenkinsfile]: Eigentlich hätten wir das durch einen sinnvollen Kommentar ersetzen können ;)

Oder ist das nicht sinnvoll?

Comment (cmg-dev) [reply _templates/Jenkinsfile]: Wenn ihr das für Sinnvoll erachtet, bitte als Issue verfassen.

Comment (Paulomart) [reply _templates/Jenkinsfile]: Ich denke nicht, dass es notwendig ist.

Comment (cmg-dev) [reply _templates/Jenkinsfile]: okay. dann lassen wir das.

**Comment by cmg-dev** [comment]
@Paulomart Bitte Konflikte lösen und dann mergen.

**Pull request merged**